### PR TITLE
chore: documentation and ci updates

### PR DESCRIPTION
**Pull request by Fdawgs**
Update example of helmet config as v5.2.0 of `fastify-helmet` adds support for CSP default directive exports

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -388,10 +389,13 @@ You can integration this plugin with ```fastify-helmet``` with some little work.
...
-        imgSrc: ["'self'", 'data:', 'validator.swagger.io'],
-        scriptSrc: ["'self'"].concat(instance.swaggerCSP.script),
-        styleSrc: ["'self'", 'https:'].concat(instance.swaggerCSP.style)
+					...helmet.contentSecurityPolicy.getDefaultDirectives(),
+					"form-action": ["'self'"],
+					"img-src": ["'self'", "data:", "validator.swagger.io"],
+					"script-src": ["'self'"].concat(instance.swaggerCSP.script),
+					"style-src": ["'self'", "https:"].concat(
+						instance.swaggerCSP.style
+					),
```
Comment (mcollina) [inline README.md:398]: why this style change?

**Review by mcollina — CHANGES_REQUESTED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (Fdawgs) [reply README.md:398]: Hi @mcollina, has to be in kebab-case unfortunately, see https://github.com/fastify/fastify-helmet/pull/109#issuecomment-759411530

Comment (mcollina) [reply README.md:398]: I mean, the alignment is all wrong.

Comment (Fdawgs) [reply README.md]: Sorry ended up mixing tabs with the spaces :confounded: , fixed in 6fae0e1

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**